mpfr fails to install on osx

I run into this in the context of installing `cgal`, which requires `mpfr`:

https://github.com/Microsoft/vcpkg/issues/3851

Starting package 81/82: mpfr:x64-osx
Building package mpfr[core]:x64-osx...
-- Using cached /Users/adam/vcpkg/downloads/mpfr-4.0.1.tar.xz
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
CMake Error at scripts/cmake/vcpkg_build_cmake.cmake:175 (message):
Command failed: /Users/adam/vcpkg/downloads/tools/cmake-3.12.4-osx/cmake-3.12.4-Darwin-x86_64/CMake.app/Contents/bin/cmake;--build;.;--config;Debug;--target;install;--;-v;-j1
Working Directory: /Users/adam/vcpkg/buildtrees/mpfr/x64-osx-dbg
See logs for more information:
/Users/adam/vcpkg/buildtrees/mpfr/install-x64-osx-dbg-out.log

Call Stack (most recent call first):
scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
ports/mpfr/portfile.cmake:18 (vcpkg_install_cmake)
scripts/ports.cmake:71 (include)

Error: Building package mpfr:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with .\vcpkg update, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
Package: mpfr:x64-osx
Vcpkg version: 2018.11.23-unknownhash

Additionally, attach any relevant sections from the log files above.
Here's the full error log:

┌─[adam][hapkido][±][master ✓][~/vcpkg/buildtrees/mpfr]
└─▪ cat install-x64-osx-dbg-out.log 
[1/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/abort_prec_max.c.o -MF CMakeFiles/mpfr.dir/src/abort_prec_max.c.o.d -o CMakeFiles/mpfr.dir/src/abort_prec_max.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/abort_prec_max.c
[2/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/add.c.o -MF CMakeFiles/mpfr.dir/src/add.c.o.d -o CMakeFiles/mpfr.dir/src/add.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/add.c
[3/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/add_d.c.o -MF CMakeFiles/mpfr.dir/src/add_d.c.o.d -o CMakeFiles/mpfr.dir/src/add_d.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/add_d.c
[4/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/add_ui.c.o -MF CMakeFiles/mpfr.dir/src/add_ui.c.o.d -o CMakeFiles/mpfr.dir/src/add_ui.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/add_ui.c
[5/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/acos.c.o -MF CMakeFiles/mpfr.dir/src/acos.c.o.d -o CMakeFiles/mpfr.dir/src/acos.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/acos.c
[6/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/acosh.c.o -MF CMakeFiles/mpfr.dir/src/acosh.c.o.d -o CMakeFiles/mpfr.dir/src/acosh.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/acosh.c
[7/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/asinh.c.o -MF CMakeFiles/mpfr.dir/src/asinh.c.o.d -o CMakeFiles/mpfr.dir/src/asinh.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/asinh.c
[8/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/agm.c.o -MF CMakeFiles/mpfr.dir/src/agm.c.o.d -o CMakeFiles/mpfr.dir/src/agm.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/agm.c
[9/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/asin.c.o -MF CMakeFiles/mpfr.dir/src/asin.c.o.d -o CMakeFiles/mpfr.dir/src/asin.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/asin.c
[10/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/add1.c.o -MF CMakeFiles/mpfr.dir/src/add1.c.o.d -o CMakeFiles/mpfr.dir/src/add1.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/add1.c
[11/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/atan2.c.o -MF CMakeFiles/mpfr.dir/src/atan2.c.o.d -o CMakeFiles/mpfr.dir/src/atan2.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/atan2.c
[12/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/add1sp.c.o -MF CMakeFiles/mpfr.dir/src/add1sp.c.o.d -o CMakeFiles/mpfr.dir/src/add1sp.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/add1sp.c
[13/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/atan.c.o -MF CMakeFiles/mpfr.dir/src/atan.c.o.d -o CMakeFiles/mpfr.dir/src/atan.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/atan.c
[14/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/ai.c.o -MF CMakeFiles/mpfr.dir/src/ai.c.o.d -o CMakeFiles/mpfr.dir/src/ai.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/ai.c
[15/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/buildopt.c.o -MF CMakeFiles/mpfr.dir/src/buildopt.c.o.d -o CMakeFiles/mpfr.dir/src/buildopt.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/buildopt.c
[16/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/atanh.c.o -MF CMakeFiles/mpfr.dir/src/atanh.c.o.d -o CMakeFiles/mpfr.dir/src/atanh.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/atanh.c
[17/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/bernoulli.c.o -MF CMakeFiles/mpfr.dir/src/bernoulli.c.o.d -o CMakeFiles/mpfr.dir/src/bernoulli.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/bernoulli.c
[18/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/check.c.o -MF CMakeFiles/mpfr.dir/src/check.c.o.d -o CMakeFiles/mpfr.dir/src/check.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/check.c
[19/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/clears.c.o -MF CMakeFiles/mpfr.dir/src/clears.c.o.d -o CMakeFiles/mpfr.dir/src/clears.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/clears.c
FAILED: CMakeFiles/mpfr.dir/src/clears.c.o 
/Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/clears.c.o -MF CMakeFiles/mpfr.dir/src/clears.c.o.d -o CMakeFiles/mpfr.dir/src/clears.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/clears.c
In file included from /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/clears.c:32:
/Library/Developer/CommandLineTools/usr/lib/clang/10.0.1/include/varargs.h:25:4: error: "Please use <stdarg.h> instead of <varargs.h>"
  #error "Please use <stdarg.h> instead of <varargs.h>"
   ^
/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/clears.c:42:2: error: expected function body after function declarator
 va_dcl
 ^
2 errors generated.
[20/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cache.c.o -MF CMakeFiles/mpfr.dir/src/cache.c.o.d -o CMakeFiles/mpfr.dir/src/cache.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cache.c
[21/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/clear.c.o -MF CMakeFiles/mpfr.dir/src/clear.c.o.d -o CMakeFiles/mpfr.dir/src/clear.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/clear.c
[22/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cbrt.c.o -MF CMakeFiles/mpfr.dir/src/cbrt.c.o.d -o CMakeFiles/mpfr.dir/src/cbrt.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cbrt.c
[23/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cmp.c.o -MF CMakeFiles/mpfr.dir/src/cmp.c.o.d -o CMakeFiles/mpfr.dir/src/cmp.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cmp.c
[24/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/beta.c.o -MF CMakeFiles/mpfr.dir/src/beta.c.o.d -o CMakeFiles/mpfr.dir/src/beta.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/beta.c
[25/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cmp2.c.o -MF CMakeFiles/mpfr.dir/src/cmp2.c.o.d -o CMakeFiles/mpfr.dir/src/cmp2.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cmp2.c
[26/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cmp_abs.c.o -MF CMakeFiles/mpfr.dir/src/cmp_abs.c.o.d -o CMakeFiles/mpfr.dir/src/cmp_abs.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cmp_abs.c
[27/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cmp_ld.c.o -MF CMakeFiles/mpfr.dir/src/cmp_ld.c.o.d -o CMakeFiles/mpfr.dir/src/cmp_ld.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cmp_ld.c
[28/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cmp_d.c.o -MF CMakeFiles/mpfr.dir/src/cmp_d.c.o.d -o CMakeFiles/mpfr.dir/src/cmp_d.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cmp_d.c
[29/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/comparisons.c.o -MF CMakeFiles/mpfr.dir/src/comparisons.c.o.d -o CMakeFiles/mpfr.dir/src/comparisons.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/comparisons.c
[30/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/constant.c.o -MF CMakeFiles/mpfr.dir/src/constant.c.o.d -o CMakeFiles/mpfr.dir/src/constant.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/constant.c
[31/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cmp_ui.c.o -MF CMakeFiles/mpfr.dir/src/cmp_ui.c.o.d -o CMakeFiles/mpfr.dir/src/cmp_ui.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cmp_ui.c
[32/236] /Library/Developer/CommandLineTools/usr/bin/cc -DHAVE_STDINT_H -D__MPFR_WITHIN_MPFR -I/Users/adam/vcpkg/installed/x64-osx/include -I. -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1 -I/Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src -fPIC -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -MD -MT CMakeFiles/mpfr.dir/src/cmp_si.c.o -MF CMakeFiles/mpfr.dir/src/cmp_si.c.o.d -o CMakeFiles/mpfr.dir/src/cmp_si.c.o   -c /Users/adam/vcpkg/buildtrees/mpfr/src/mpfr-4.0.1/src/cmp_si.c
ninja: build stopped: subcommand failed.

--- a/ports/mpfr/CMakeLists.txt
+++ b/ports/mpfr/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.3.0)
 
-project(MPFR VERSION 3.1.5 LANGUAGES C)
+project(MPFR VERSION 4.0.2 LANGUAGES C)
 
 # Find GMP (or MPIR)
 find_library(GMP_LIBRARY NAMES gmp mpir)
@@ -259,9 +259,9 @@ if(BUILD_SHARED_LIBS)
 endif()
 
 try_compile(I_HAVE_STDARG ${CMAKE_BINARY_DIR} ${PROJECT_SOURCE_DIR}/test_stdarg.c)
-if (I_HAVE_STDARG)
+if (${I_HAVE_STDARG})
     target_compile_definitions(mpfr PRIVATE HAVE_STDARG)
-endif (I_HAVE_STDARG)
+endif (${I_HAVE_STDARG})
 
 target_link_libraries(mpfr ${GMP_LIBRARIES})
 

--- a/ports/mpfr/CMakeLists.txt
+++ b/ports/mpfr/CMakeLists.txt
@@ -254,14 +254,10 @@ add_library(mpfr ${SRCS})
 # target_compile_definitions(mpfr PRIVATE HAVE_CONFIG_H)
 target_compile_definitions(mpfr PRIVATE __MPFR_WITHIN_MPFR)
 target_compile_definitions(mpfr PRIVATE HAVE_STDINT_H)
+target_compile_definitions(mpfr PRIVATE HAVE_STDARG)
 if(BUILD_SHARED_LIBS)
     target_compile_definitions(mpfr PRIVATE __GMP_LIBGMP_DLL)
 endif()
-
-try_compile(I_HAVE_STDARG ${CMAKE_BINARY_DIR} ${PROJECT_SOURCE_DIR}/test_stdarg.c)
-if (${I_HAVE_STDARG})
-    target_compile_definitions(mpfr PRIVATE HAVE_STDARG)
-endif (${I_HAVE_STDARG})
 
 target_link_libraries(mpfr ${GMP_LIBRARIES})
 

--- a/ports/mpfr/CONTROL
+++ b/ports/mpfr/CONTROL
@@ -1,5 +1,5 @@
 Source: mpfr
-Version: 4.0.1
+Version: 4.0.2
 Homepage: https://www.mpfr.org
 Description: The MPFR library is a C library for multiple-precision floating-point computations with correct rounding
 Build-Depends: mpir

--- a/ports/mpfr/CONTROL
+++ b/ports/mpfr/CONTROL
@@ -1,5 +1,5 @@
 Source: mpfr
-Version: 4.0.2
+Version: 4.0.2-1
 Homepage: https://www.mpfr.org
 Description: The MPFR library is a C library for multiple-precision floating-point computations with correct rounding
 Build-Depends: mpir

--- a/ports/mpfr/portfile.cmake
+++ b/ports/mpfr/portfile.cmake
@@ -1,9 +1,9 @@
 include(vcpkg_common_functions)
-set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/mpfr-4.0.1)
+set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/mpfr-4.0.2)
 vcpkg_download_distfile(ARCHIVE
-    URLS "http://www.mpfr.org/mpfr-4.0.1/mpfr-4.0.1.tar.xz"
-    FILENAME "mpfr-4.0.1.tar.xz"
-    SHA512 137ad68bc1e33a155edc1247fcdba27f999cf48ed526773136584090ddf2cfdfc9ea79fbf74ea1943b835b4b1ff29b05087114738c6ad3b485848540f30cac4f
+    URLS "http://www.mpfr.org/mpfr-4.0.2/mpfr-4.0.2.tar.xz"
+    FILENAME "mpfr-4.0.2.tar.xz"
+    SHA512 d583555d08863bf36c89b289ae26bae353d9a31f08ee3894520992d2c26e5683c4c9c193d7ad139632f71c0a476d85ea76182702a98bf08dde7b6f65a54f8b88
 )
 
 vcpkg_extract_source_archive(${ARCHIVE})

--- a/ports/mpfr/portfile.cmake
+++ b/ports/mpfr/portfile.cmake
@@ -8,7 +8,6 @@ vcpkg_download_distfile(ARCHIVE
 
 vcpkg_extract_source_archive(${ARCHIVE})
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
-file(COPY ${CMAKE_CURRENT_LIST_DIR}/test_stdarg.c DESTINATION ${SOURCE_PATH})
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}

--- a/ports/mpfr/test_stdarg.c
+++ b/ports/mpfr/test_stdarg.c
@@ -1,6 +1,5 @@
-# include <stdarg.h>
+#include <stdarg.h>
 
 int main(int argc, char *argv) {
     return 0;
 }
-

--- a/ports/mpfr/test_stdarg.c
+++ b/ports/mpfr/test_stdarg.c
@@ -1,5 +1,0 @@
-#include <stdarg.h>
-
-int main(int argc, char *argv) {
-    return 0;
-}

